ui/page/top/view.jsx

```diff
@@ -7,20 +7,24 @@ import ClaimEffectiveAmount from 'component/claimEffectiveAmount';
 import SearchTopClaim from 'component/searchTopClaim';
 import { ORDER_BY_TOP, FRESH_ALL } from 'constants/claim_search';
 import Button from 'component/button';
+import I18nMessage from 'component/i18nMessage';
+import * as PAGES from 'constants/pages';
 
 type Props = {
   name: string,
+  beginPublish: string => void,
 };
 
 function TopPage(props: Props) {
-  const { name } = props;
+  const { name, beginPublish } = props;
   const [channelActive, setChannelActive] = React.useState(false);
 
+  const queryName = name[0] === '@' ? name.slice(1) : name;
   return (
     <Page>
       <SearchTopClaim query={name} hideLink setChannelActive={setChannelActive} />
       <ClaimListDiscover
-        name={channelActive ? `@${name}` : name}
+        name={channelActive ? `@${queryName}` : queryName}
```

Comment:
Looking at the `queryName` line above... I think you can just do `name={name}`?

Reply:
Tried it, can't do it - it doesn't switch what claimListDiscover searches that way.